Makefile

```diff
@@ -28,7 +28,7 @@ codecheck: $(PYSRC)
 errcheck: $(PYSRC)
 	-echo "Running check for errors only"
 	pyflakes $(PYSRC)
-	pylint -E --rcfile=.pylint $(PYSRC)
```

Comment:
Why disable lint?

Reply:
> Why disable lint?

changed it back not to disable lint. I had vague memory when I installed it several months ago I may have some issue related to it, for simplicity I just disabled this lint back then.